ecs_fargate/manifest.json

```diff
@@ -5,7 +5,7 @@
     "orchestration"
   ],
   "creates_events": false,
-  "display_name": "Amazon Fargate",
+  "display_name": "Amazon ECS on AWS Fargate",
```

Comment:
I don't think this display_name field can be changed. cc @nmuesch if you have more context on renaming tiles

Reply:
Indeed, the display_name can't currently be changed, however you can change the `public_title`.